wa/workloads/googlemaps/__init__.py

```diff
@@ -0,0 +1,73 @@
+from wa import Parameter, ApkUiautoWorkload
+
+
+class GoogleMaps(ApkUiautoWorkload):
+
+    name = 'googlemaps'
+    description = '''
+    A workload to perform standard navigation tasks with Google Maps. This workload searches
+    for known locations, pans and zooms around the map, and follows driving directions
+    along a route.
+
+    To run the workload in offline mode, 'databases.tar' and 'files.tar' archives are required.
+    In order to generate these files, Google Maps should first be operated from an
+    Internet-connected environment, and a region around Cambridge, England should be downloaded
+    for offline use. This region must include the landmarks used in the UIAutomator program,
+    which include Cambridge train station and Corpus Christi college.
+
+    Following this, the files of interest can be found in the 'databases' and 'files'
+    subdirectories of the '/data/data/com.google.android.apps.maps/' directory. The contents of
+    these subdirectories can be archived into tarballs using commands such as
+    ``tar -cvf databases.tar -C /path/to/databases .``. These 'databases.tar' and 'files.tar' archives
+    should then be placed in the '~/.workload_automation/dependencies/googlemaps' directory on your
+    local machine, creating this if it does not already exist.
+
+    Known working APK version: 9.72.2
+    '''
+    package_names = ['com.google.android.apps.maps']
+
+    parameters = [
+        Parameter('offline_mode', kind=bool, default=False, description='''
+                  If set to ``True``, the workload will execute in offline mode.
+                  This mode requires root and makes use of a tarball of database
+                  files 'databases.tar' and a tarball of auxiliary files 'files.tar'.
+                  These tarballs are extracted directly to the application's 'databases'
+                  and 'files' directories respectively in '/data/data/com.google.android.apps.maps/'.
+                  '''),
+    ]
+
+    @property
+    def requires_network(self):
+        return not self.offline_mode
+
+    @property
+    def requires_rerun(self):
+        # In offline mode we need to restart the application after modifying its data directory
+        return self.offline_mode
+
+    def __init__(self, target, **kwargs):
+        super(GoogleMaps, self).__init__(target, **kwargs)
+        if self.offline_mode:
+            self.deployable_assets = ['databases.tar', 'files.tar']
+            self.clean_assets = True
```

Comment:
This parameter should be `self.cleanup_assets`

Reply:
Oh, really? `clean_assets` is being used in the Gmail and Chrome workloads too (I copied it across from the original Gmail workload file if I remember correctly)

Reply:
Yea, it was a mistake in the original workload. I've submitted a PR to correct it in those other workloads.